core/src/test/scala/org/apache/spark/FileSuite.scala

```diff
@@ -94,6 +94,14 @@ class FileSuite extends SparkFunSuite with LocalSparkContext {
     assert(compressedFile.length < normalFile.length)
   }
 
+  test("text files do not allow null rows") {
+    sc = new SparkContext("local", "test")
+    val outputDir = new File(tempDir, "output").getAbsolutePath
+    val nums = sc.makeRDD((1 to 100) ++ Seq(null))
+    val exception = intercept[SparkException](nums.saveAsTextFile(outputDir))
+    assert(Utils.exceptionString(exception).contains("should not contain null rows"))
```

Comment:
Just write `exception .getMessage.contains...`?

Reply:
@maropu Here, `exception.getMessage` can not get the root cause message, but returns `Job aborted`, so I have to use `Utils.exceptionString`.

Reply:
ah, I see.